app/src/main/java/com/github/libretube/helpers/LocaleHelper.kt

```diff
@@ -29,7 +28,7 @@ object LocaleHelper {
 
     fun updateLanguage(context: Context) {
         val locale = getAppLocale()
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) updateResources(context, locale)
```

Comment:
Wasn't this meant to be an if else check, or is it intentional to always call updateResourcesLegacy?

Reply:
I think so too, `updateResourcesLegacy` can probably be removed now.